source/apiVolontaria/apiVolontaria/serializers.py

```diff
@@ -111,6 +113,16 @@ class Meta:
         validators=[phone_number],
     )
 
+    managed_cell = serializers.SerializerMethodField()
```

Comment:
Here you can use a [nested serializer](http://www.django-rest-framework.org/api-guide/relations/#nested-relationships) directly and use the `related_name` of the user model so you don't need to add some logic.

https://github.com/Volontaria/API-Volontaria/blob/c592a5b264220afe709197fc8ab6f3908892414c/source/apiVolontaria/volunteer/models.py#L150